src/main/java/sirius/kernel/xml/StructuredOutput.java

```diff
@@ -38,14 +38,6 @@
      */
     void endResult();
 
-    /**
-     * Starts a new object with the given name.
-     *
-     * @param name the name of the element to start
-     * @return the output itself for fluent method calls
-     */
-    StructuredOutput beginObject(@Nonnull String name);
```

Comment:
sure about this? this would be a breaking change - deprecate otherwise?

Reply:
It has an overload `StructuredOutput beginObject(@Nonnull String name, Attribute... attributes)` which will be called instead. A library user might need to recompile, but the code will still be compatible